### PR TITLE
make topics collapsable

### PR DESCRIPTION
**Pull request by ortutay**

### CODE REVIEW

File: articles/templates/articles/base.html [hunk 1]
```diff
@@ -13,29 +13,80 @@
...
+
+           .title {
+             font-size: 24px;
+             font-weight: bold;
+           }
+          </style>
+           <script type="text/javascript">
+           $(document).ready(function() {
+             $('.toggle').click(function() {
+               let that = this;
```
Comment (ygreif) [inline articles/templates/articles/base.html:76]: This is poetic

Comment (ortutay) [reply articles/templates/articles/base.html:76]: heh this is actually a pretty common convention

File: articles/templates/articles/topic.html [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-          {% include filename with root=child %}
-          {% endwith %}
-      {% endfor %}
+  <div class="toggle-wrapper">
+    <div class="row margin-b-2">
+      <div class="toggle">
+        <div class="toggle-icon"><i class="fas fa-caret-square-down"></i></div>
+        <div class="title" id="{{root.title}}"> {{ root.title }} </div>
+      </div>
+      <div class="toggle-content">
```
Comment (ygreif) [inline articles/templates/articles/topic.html:8]: Is there something here that makes the toggle-content hidden vs not hidden?

Comment (ortutay) [reply articles/templates/articles/topic.html:8]: yea it's in base.html

**Pull request merged**